org.gnome.Photos.json

```diff
@@ -294,13 +294,14 @@
                 "-Dminer_fs=true",
                 "-Dminer_fs_cache_location=$XDG_CACHE_HOME/gnome-photos/miner/files",
                 "-Dminer_rss=false",
+		"-Dbattery_detection=none",
```

Comment:
There is a new option here https://gitlab.gnome.org/GNOME/tracker-miners/-/blob/master/meson_options.txt?ref_type=heads#L22. I will ask what it does.

Reply:
I added this because otherwise the build fail.

Reply:
fixed the indentation however. Emacs started adding tabs in JSON and it drives me nuts.